api/v1/swagger.yaml

```diff
@@ -1174,6 +1174,590 @@ paths:
           $ref: "#/responses/genericError"
 
   # ---------------------------------------------------------------------------
+  /collectors:
+    post:
+      security:
+        - jwt: []
+      tags: [ collectors ]
+      summary: Create a new collector
+      description: >-
+        Create a new collector. This request may only be issued by the refocus-collector register command.
+        The new record is automatically write-protected for just the user who created it. The default status: STOPPED.
+        isRegistered is true. System fills in the timeStamp fields.
+      operationId: postCollector
+      parameters:
+        -
+          name: queryBody
+          description: Request body.
+          in: body
+          required: true
+          schema:
+            type: object
+            description: A collector of observation which can be measured.
+            properties:
+              name:
+                type: "string"
+                description: "Should be unique."
+              status:
+                type: "string"
+                description: "The collector status."
+                enum:
+                - "STOPPED"
+                - "RUNNING"
+                - "PAUSED"
+              description:
+                type: string
+                maxLength: 4096
+                description: >
+                  Describes this Collector; helps a user determine which Collectors they should assign to run their collection.
```

Comment:
so the description field of a collector cannot be modified by any collector, other than itself?